App.js

```diff
@@ -192,11 +194,11 @@ export default class App extends Component {
       })
   }
 
-  onWebSocketMessage(event) {
+  onWebSocketMessage = (event) => {
```

Comment:
You're doing thius change to have `this` automatically binded that's it? (ES6-syntax)

Not sure if it's ok to mix two styles for method declarations, maybe we should update all methods?

Reply:
> You're doing thius change to have this automatically binded that's it? (ES6-syntax)

Yes, that's the only reason.

> Not sure if it's ok to mix two styles for method declarations, maybe we should update all methods?

The alternative is to bind `onWebSocketMessage` in the `constructor`. Whichever you prefer!

Reply:
Let's keep binding in the `constructor` for now, for consistence.